docs/text_summaries.ipynb

```diff
@@ -0,0 +1,327 @@
+        "  tf.summary.text(\"My first text\", my_text, step=0)"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "rngALbRogXe6"
+      },
+      "source": [
+        "Now, use TensorBoard to examine the image. Wait a few seconds for the UI to spin up."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "T_X-wIy-lD9f"
+      },
+      "outputs": [],
+      "source": [
+        "%tensorboard --logdir logs"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "c8n8YqGlT3-c"
+      },
+      "source": [
+        "<!-- <img class=\"tfo-display-only-on-site\" src=\"https://github.com/tensorflow/tensorboard/blob/master/docs/images/images_single.png?raw=1\"/> -->"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "34enxJjjgWi7"
+      },
+      "source": [
+        "## Organizing multiple text streams\n",
+        "\n",
+        "If you have multiple streams of text, you can keep them in separate namespaces to help organize them, just like scalars or other data.\n",
+        "\n",
+        "Note that if you log text at many steps, TensorBoard will subsample the steps to display.  Again, this behavior is the same as for scalars, where the data is subsampled to make the presentation manageable."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "dda6960f0119"
+      },
+      "outputs": [],
+      "source": [
+        "# Sets up a second directory to not overwrite the first one.\n",
+        "logdir = \"logs/multiple_texts/\" + datetime.now().strftime(\"%Y%m%d-%H%M%S\")\n",
+        "# Creates a file writer for the log directory.\n",
+        "file_writer = tf.summary.create_file_writer(logdir)\n",
+        "\n",
+        "# Using the file writer, log the text.\n",
+        "with file_writer.as_default():\n",
+        "  with tf.name_scope(\"Name scope 1\"):\n",
+        "    for step in range(20):\n",
+        "      tf.summary.text(\"a stream of text\", f\"Hello from step {step}\", step=step)\n",
+        "      tf.summary.text(\"another stream of text\", f\"This can be kept separate {step}\", step=step)\n",
+        "  with tf.name_scope(\"Name scope 2\"):\n",
+        "    tf.summary.text(\"just from step 0\", f\"This is an important announcement from step 0\", step=0)\n",
+        "    "
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "515199f4b547"
+      },
+      "outputs": [],
+      "source": [
+        "%tensorboard --logdir logs"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "bjACE1lAsqUd"
+      },
+      "source": [
+        "## Using markdown\n",
+        "\n",
+        "Tensorboard supports logging text in markdown to make it easier to read and understand."
```

Comment:
Technically, you actually don't have a choice: TensorBoard interprets all text that users log as potential Markdown.  You might expect this is undesirable sometimes and that would be correct: https://github.com/tensorflow/tensorboard/issues/830

Until we fix that, I think it's perhaps more accurate to title this section `## Markdown interpretation` and then say something like:

> TensorBoard interprets text summaries as Markdown, since rich formatting can make the data you log easier to read and understand, as shown below.  (If you don't want Markdown interpretation, see [this issue](https://github.com/tensorflow/tensorboard/issues/830) for workarounds to suppress interpretation.)

Reply:
done, yes also better, thanks.